### PR TITLE
Essential read

### PR DESCRIPTION
**Pull request by zeftilldeath**
Better late than never, added some design/ux love that I would have added earlier if I hadn't been on holiday :dancer: Thanks @Jholder112233 

Also a bonus copy change for @stephanfowler 

Before
screen shot 2015-11-26 at 10 31 17

After
screen shot 2015-11-26 at 10 31 22

New tablet breakpoint
screen shot 2015-11-26 at 10 31 42

### CODE REVIEW

**Comment by Jholder112233** [comment]
Code looks good if you're happy across browsers/breakpoints! :+1: :ship:

**Comment by paperboyo** [comment]
This "bigger image". Is it our standard 5:3 ratio crop?

Also [triangles](https://camo.githubusercontent.com/4da270382cd1fb8af24a2e75c8d46ba4ff241bea/68747470733a2f2f662e636c6f75642e6769746875622e636f6d2f6173736574732f38353738332f3831313337332f37323662616161342d656564322d313165322d393530652d3863396231353732613832662e504e47) long time no see... :)

File: static/src/stylesheets/module/onward/_essential-read.scss [hunk 1]
```diff
@@ -3,8 +3,14 @@
     margin-bottom: $gs-baseline;
 
     .fc-container__header__title {
+        font-size: 2.25rem;
```
Comment (sndrs) [inline static/src/stylesheets/module/onward/_essential-read.scss]: @zeftilldeath these should use the guss typography mixins (sorry, missed this PR!)

**Comment by zeftilldeath** [comment]
@paperboyo Yep the image is 5:3.  
@sndrs Good spot, was in a rush to get these changes out as there's only a couple days left of the test. Will update this afternoon.

**Comment by paperboyo** [comment]
@zeftilldeath Supa! Thx.

**Comment by sndrs** [comment]
@zeftilldeath sorry to be picky - just if it goes well we need to be sure they don't end up left in there

**Pull request merged**